RELEASE_NOTES.md

```diff
@@ -0,0 +1,6 @@
+# Release notes
+
+## [x.x.x] - 202x-xx-xx
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=5870223)